src/main/java/com/starl0stgaming/gregicalitystarbound/api/space/planets/worldgen/PlanetBiome.java

```diff
@@ -0,0 +1,118 @@
+package com.starl0stgaming.gregicalitystarbound.api.space.planets.worldgen;
+
+import com.google.gson.JsonObject;
+import com.starl0stgaming.gregicalitystarbound.api.space.dimensions.world.DummyBiome;
+import com.starl0stgaming.gregicalitystarbound.api.util.StringUtil;
+import net.minecraft.block.Block;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.biome.Biome;
+
+import net.minecraft.world.biome.Biome.BiomeProperties;
+import net.minecraftforge.fml.common.registry.ForgeRegistries;
+
+public class PlanetBiome {
+    // Both of these strings get turned into blocks
+    private IBlockState fillerBlock;
+    private IBlockState topBlock;
```

Comment:
Shit I forgot, same as WorldGenDetails, need a string version of each of these too for gson, just call them filler and top respectively